code/modules/mob/living/carbon/human/inventory.dm

```diff
@@ -79,31 +79,47 @@
 
 //This is an UNSAFE proc. Use mob_can_equip() before calling this one! Or rather use equip_to_slot_if_possible() or advanced_equip_to_slot_if_possible()
 // Initial is used to indicate whether or not this is the initial equipment (job datums etc) or just a player doing it
-/mob/living/carbon/human/equip_to_slot(obj/item/I, slot, initial = FALSE, redraw_mob = FALSE)
+/mob/living/carbon/human/equip_to_slot(obj/item/I, slot, initial = FALSE, redraw_mob = FALSE, force_equip = FALSE)
 	if(!..()) //a check failed or the item has already found its slot
 		return
 
 	var/not_handled = FALSE //Added in case we make this type path deeper one day
 	switch(slot)
 		if(ITEM_SLOT_BELT)
 			if(belt)
-				return
+				if(!force_equip)
```

Comment:
I'm not comfortable with expanding `equip_to_slot` even more--you should make this behavior its own proc that calls `equip_to_slot`.